content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,250 @@
+    * logs:CreateLogGroup
+    * logs:DescribeLogGroups
+    * logs:PutRetentionPolicy
+    * s3:CreateBucket
+    * s3:GetObject
+    * s3:GetObjectVersion
+    * secretsmanager:CreateSecret
+    * secretsmanager:GetSecretValue
+    * secretsmanager:PutSecretValue
+    * serverless:CreateCloudFormationTemplate
+
+
+## Setup
+
+
+2. From the AWS tile on the [Integrations page][8] in your Datadog account, select the Datadog products you wish to integrate with this AWS Account. This will select the right default settings for integrating data for those products. These settings can always be changed in the future if needed.
+{{< img src="getting_started/integrations/cloudformation-setup.png" alt="An image from the Datadog AWS integration tile showing the options for establishing the integration. The Role Delegation tab is highlighted.">}}
+
+3. Select the AWS Region where the CloudFormation stack will be launched and the Datadog Lambda forwarder for sending AWS logs to Datadog is created (if you selected Log Management).
+
+    **Note**: CloudWatch metrics are collected from ALL AWS regions you are using regardless of the region you select.
+
+4. Select or create the Datadog API Key used to send data from your AWS account to Datadog.
+
+5. Click "Launch CloudFormation Template".
+
+6. This opens the AWS Console and loads the CloudFormation stack. All the parameters will be correctly filled in based on your selections in the prior Datadog form, so you do not need to edit those unless desired.
+**Note:** The `DatadogAppKey` parameter enables the CloudFormation stack to add and edit the Datadog configuration for this AWS account via API calls to Datadog. This is automatically generated and tied to your Datadog account.
+{{< img src="getting_started/integrations/params.png" alt="An image from the AWS CloudFormation create-stack page that shows the Stack name as datadog, IAMRoleName as DatadogIntegrationRole, ExternalId as an obfuscated value ending in be46, DdApiKey as an obfuscated value.">}}
+
+7. Check the required boxes from AWS and click `Create stack`:
+{{< img src="getting_started/integrations/cloudformation-options-2.png" alt="An image from the AWS CloudFormation create-stack page that shows the Advanced options of creating the Datadog stack. DdAWSAccountId parameter is filled in with 464622532012, DdForwarderName parameter is filled in with DatadogForwarder, and InstallDatadogPolicyMacro is set as true. Below these parameters is a Capabilities section with two checkboxes, both of which are checked. The first checkbox states 'I acknowledge that AWS CloudFormation might create IAM resources with custom names.' The second checkbox states 'I acknowledge that AWS CloudFormation might require the following capability: CAPABILITY_AUTO_EXPAND'.">}}
+
+    This begins the creation process for the Datadog stack along with three nested stacks. This could take several minutes; ensure that the stack is successfully created before proceeding:
```

Comment:
```suggestion
    This launches the creation process for the Datadog stack along with three nested stacks. This could take several minutes. Ensure that the stack is successfully created before proceeding:
```